KA-Flocking/Assets/Scripts/BehaviourScripts/AlignmentBehaviour.cs

```diff
@@ -19,14 +19,15 @@ public class AlignmentBehaviour : FilteredFlockBehaviour
             FlockAgent a = item.GetComponentInParent<FlockAgent>();
             distanceSqr = Vector3.SqrMagnitude(item.position - agent.transform.position);
             if (a.unit.GetType().ToString().Equals("Scout") ) {
-                aligntmentMove += 20*item.transform.forward/distanceSqr;
 
-            } else {
-                aligntmentMove += item.transform.forward/distanceSqr;
+                if (distanceSqr != 0) aligntmentMove += 20*item.transform.forward/distanceSqr;

```

Comment:
distanceSqr should not be able to be 0, that'd mean that two different units are on the same location, unsure how this could arise, but if It happens I guess we can keep it